examples/misc/bin/dcat_tooltips.html

```diff
@@ -0,0 +1,53 @@
+<!--
+  Tooltip data.
+
+  Source file: dcat.dart
+
+  Ensure that:
+  - Each //!tip() entry in the source file has a corresponding entry below (an
+    entry with a `name` attribute that matches the //!tip() text), in the same
+    order as in the source file.
+  - Each entry below has a corresponding //!tip() entry in the source file.
+-->
+<ul class="fp-tooltips" style="display: none">
+  <li name="List<String> arguments" title="Command-line arguments">
+    Command-line arguments are passed in by the system when the program starts.
+  </li>
+  <li name="argResults = parser.parse(arguments)" title="Arguments parser">
+    The <code>ArgParser</code> class provides help with parsing command-line arguments.
+  </li>
+  <li name="async" title="Async function">
+    An asynchronous function marked with <code>async</code> that returns a future.
```

Comment:
I know that this update changes the meaning of the tooltip, but I wasn't sure how else to updated it. Let me know what you think.

Reply:
Hmmm... "that returns a future" implies that there's another kind of async function. Maybe just change it to "A function marked with `async`."? Or:

A function that has a body marked as `async`.  These functions are asynchronous and return futures.

Reply:
How about: _A function marked as `async`. These functions are asynchronous and return futures._ (In contrast to: A function that has a body marked as `async`. ...)

Reply:
I always get a little squirrely about what `async` modifies, but my definition was precise at the expense of readability. Yours is better.